block/manager.go

```diff
@@ -738,16 +738,15 @@ func getRemainingSleep(start time.Time, interval, defaultSleep time.Duration) ti
 	return sleepDuration
 }
 
-func (m *Manager) getCommit(header types.Header) (*types.Commit, error) {
+func (m *Manager) getSignature(header types.Header) (*types.Signature, error) {
 	// note: for compatibility with tendermint light client
 	consensusVote := header.MakeCometBFTVote()
 	sign, err := m.sign(consensusVote)
 	if err != nil {
 		return nil, err
 	}
-	return &types.Commit{
-		Signatures: []types.Signature{sign},
-	}, nil
+	signature := types.Signature(sign)
+	return &signature, nil
```

Comment:
[nit] no need for new variable. just `return &types.Signature(sign), nil` is good.

Reply:
<img width="988" alt="image" src="https://github.com/rollkit/rollkit/assets/16928547/699434ec-0b79-4d97-bb42-5492b4fbc34d">

I tried that, this is what I get.

Reply:
yea, my bad. you are correct.